[2024-01-23T18:41:28.773+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data manual__2024-01-23T18:33:49.992477+00:00 [queued]>
[2024-01-23T18:41:28.890+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data manual__2024-01-23T18:33:49.992477+00:00 [queued]>
[2024-01-23T18:41:28.903+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-23T18:41:29.720+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2024-01-23 18:33:49.992477+00:00
[2024-01-23T18:41:29.765+0000] {standard_task_runner.py:60} INFO - Started process 4106 to run task
[2024-01-23T18:41:29.788+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'manual__2024-01-23T18:33:49.992477+00:00', '--job-id', '494', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmpriyiv8gr']
[2024-01-23T18:41:29.818+0000] {standard_task_runner.py:88} INFO - Job 494: Subtask get_stackoverflow_data
[2024-01-23T18:41:30.212+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data manual__2024-01-23T18:33:49.992477+00:00 [running]> on host 9924936b75b5
[2024-01-23T18:41:31.098+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-23T18:33:49.992477+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-23T18:33:49.992477+00:00'
[2024-01-23T18:41:32.463+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-23T18:41:32.528+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20240123T183349, start_date=20240123T184128, end_date=20240123T184132
[2024-01-23T18:41:32.890+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T18:41:33.706+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
